docs/repo/dependencies-node-roadmap.md

```diff
@@ -26,7 +28,13 @@ The [`IDependenciesSnapshotProvider`][2] is responsible for providing access to
 
 Much of the code for the Dependencies node is concerned with creating [`IDependency`][4]s and translating them into new `IProjectTree`s when they change.
 
-## DependenciesProjectTreeProvider
+## CPS/Project System Interaction
+
+In general, items _directly_ referenced by the project file (e.g., through `Reference`, `ProjectReference`, and `Analyzer` items in the project file) are represented with `IProjectTree` nodes as opposed to `GraphNodes`. This makes it possible for them to the represented in the project's `IVsHierarchy` and, crucially, makes it easier to code the sorts of interactions users expect for these items. For example, a user should be able to right-click on an assembly reference and remove it, or modify the properties of an assembly refrence.
```

Comment:
Minor grammatical problem around _for them to the represented_. Also a typo in _refrence_.